ecosystem/sep-0009.md

```diff
@@ -50,6 +50,7 @@ Name | Type | Description
 `bank_account_number` | string | Number identifying bank account
 `bank_number` | string | Number identifying bank in national banking system (routing number in US)
 `bank_phone_number` | string | Phone number with country code for bank
+`bank_branch_number` | string | Number identifying bank branch
```

Comment:
Unless branch identifiers are _always_ numbers, I would make this field name more generic:
```suggestion
`bank_branch_identifier` | string | An identifier for the bank branch
```
If branch _numbers_ are the standard then this suggestion can be ignored.

Reply:
`bank_branch_identifier` Affin Bank 
An identifier for the. Mohd Faizal Bin Julung 
typically a branch
number. Bank 205070110343